WorkoutGenerator/PreviewWorkoutView.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  PreviewWorkoutView.swift
+//  WorkoutGenerator
+//
+//  Created by Olivia Schotz on 5/21/24.
+//
+
+import Foundation
+import SwiftUI
+import Lottie
+
+struct PreviewWorkoutView: View {
```

Comment:
todo: add comments

new preview view so users can see the proposed generated workout before saving

Reply:
sweet